data_profiler/data_readers/csv_data.py

```diff
@@ -236,8 +237,8 @@ def is_match(cls, file_path, options=None):
         max_count_percent = count_percent[max_count_index]
         
         # Infered the file was a CSV
-        if ((max_count_value > 0 or delimiter is None) \
-            and max_count_percent >= 0.8):
+        if (max_count_value > 0 or delimiter is None) \
+            and (max_count_percent >= 0.80):
```

Comment:
Would you indent this one over? Will make it easier to read.

Reply:
Done, please re-review